src/progress-steps/styled-components.js

```diff
@@ -0,0 +1,242 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+
+import {styled} from '../styles/index';
+import type {
+  StyledProgressStepsPropsT,
+  StyledStepPropsT,
+  StyledIconPropsT,
+  StyledInnerIconPropsT,
+  StyledContentPropsT,
+  StyledContentTitlePropsT,
+  StyledContentTailPropsT,
+  StyledContentDescriptionPropsT,
+  StyledNumberIconPropsT,
+  StyledNumberContentTailPropsT,
+  StyledNumberStepPropsT,
+} from './types';
+
+export const StyledProgressSteps = styled(
+  'div',
+  ({$theme}: StyledProgressStepsPropsT) => {
+    return {
+      padding: $theme.sizing.scale300,
```

Comment:
Let's use longhand style props instead of shorthand:
`paddingTop`
`paddingBottom`
...